### ISSUE TITLE
How to use printer?

### ISSUE THREAD

**Issue by ghost** [aria1th/litematica-printer#68]
I installed this mod, but I don't understand how to use it. I found your YouTube channel and somehow everything works there, but I don’t understand how. There is no "Enable Printing" button in the mod, there are just a million settings that you don't know how to work with. Can you make a guideline on how to make the printer work? It’s just that everything is so simple on your YouTube channel, but how to use it?

**Comment by nickgreenway** [comment]
Yeah, I've been trying to figure it out for an hour now. When I press caps lock nothing happens.

**Comment by PiggiesGoSqueal** [comment]
I am having this exact issue..

**Comment by PiggiesGoSqueal** [comment]
> I installed this mod, but I don't understand how to use it. I found your YouTube channel and somehow everything works there, but I don’t understand how. There is no "Enable Printing" button in the mod, there are just a million settings that you don't know how to work with. Can you make a guideline on how to make the printer work? It’s just that everything is so simple on your YouTube channel, but how to use it?

The original version of this mod on github has more information on how to use:
https://github.com/aleksilassila/litematica-printer

However, I tried both the versions on this Github and the versions on Modrinth:
https://modrinth.com/mod/litematica-printer-easyplace-extension/versions

Neither worked when pressing CAPS_LOCK or V to toggle printing. Reviewed settings in M + C as well but didn't see any reason it shouldn't work. The dev also seems intent on not answering "How To" questions in the Github so... I guess I'll find a different mod.

**Comment by ESEAbsolute** [comment]
> [NeGenaCidarmyan](/NeGenaCidarmyan)

@PiggiesGoSqueal @nickgreenway 

The version I used is 1.21, using mod version 7.2.2.

This mod is TOTALLY DIFFERENT from aleksilassila's printer. That one places one block at a time, and use a button to toggle auto printing. It DO NOT need you to hold the "PLACE" key (usually Right Mouse Button) to perform printing operation.

This is how this printer works: When the config "printerOff" is disabled, and holding the "PLACE" key, this mod automatically place multiple blocks nearby. You can consider this mod as an enhancement of "EasyPlace" to be an "EasyPlaceAura".
